### ISSUE TITLE
Can not catch user input on pages bigger than the default pageSize

### ISSUE THREAD

**Issue by PhilipTsv** [onechiporenko/ember-models-table#179]
Hi @onechiporenko ,
thanks for the nice project! 
I'm having some trouble: in didInsertElement() I'm trying to catch the number of times a user hits Tab while filling a column. (e.g. alert when they've moved down 10 rows).
This works fine with the default pageSize (10), but once you change the pageSize, it can't pick up Tabs in the newly generated rows (if you changed pageSize to 50, it only registers key presses in rows 1-10 and then nothing in 11-50).
Note that in the browser, the tab still jumps to the next row, however ember does not recognize that.
I also have a feature that enables/disables some of the columns, and once a column changes it's state, again ember does not pick up the tabs.
So to sum it up, my component isn't aware that I've extended/changed the table.
Any idea why this is happening and how can I go around it?

**Comment by PhilipTsv** [comment]
Worked around the issue by using on focus events instead of tabs.

`focus-in=(action "focus" record.name record.number)`
